config/default.yml

```diff
@@ -7,3 +7,9 @@ Minitest/AssertNil:
   StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-nil'
   Enabled: true
   VersionAdded: '0.1'
+
+Minitest/RefuteNil:
+  Description: 'Check if your test uses `refute_nil` instead of `refute_equal(nil, something)`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#refute-nil'
+  Enabled: true
+  VersionAdded: '0.1.1'
```

Comment:
I overlooked the version and updated it by https://github.com/rubocop-hq/rubocop-minitest/commit/b9c8da6. FYI, Minor version will be bumped when new cop are added.